config.yaml

```
@@ -18,29 +18,29 @@ carpentry: 'incubator'
 carpentry_description: 'Lesson Description'
 
 # Overall title for pages.
-title: 'Lesson Title' # FIXME
+title: 'Pumpkin Muffins'
```

I think we should add chocolate chips. Everyone loves them.
```suggestion
title: 'Pumpkin Chocolate Chip Muffins'
```